clang/test/ClangScanDeps/strip-llvm-args.m

```diff
@@ -0,0 +1,49 @@
+// RUN: rm -rf %t
+// RUN: split-file %s %t
+// RUN: sed -e "s|DIR|%/t|g" %t/cdb1.json.template > %t/cdb1.json
+
+// RUN: clang-scan-deps -compilation-database %t/cdb1.json -format experimental-full > %t/result1.txt
+// RUN: FileCheck %s -input-file %t/result1.txt
+
+// Verify that secondary actions get stripped, and that there's a single version
+// of module A.
+
+// CHECK:        "modules": [
+// CHECK-NEXT:     {
+// CHECK:            "command-line": [
+// CHECK-NOT:          "-treat-scalable-fixed-error-as-warning"
```

Comment:
I'd expect this to check for "-mllvm -stackmap-version=2" from the TU invocation, what's the reason for checking "-treat-scalable-fixed-error-as-warning" instead?

Reply:
Updated the test to check for `-mllvm` instead.